### PR TITLE
Starting code

### PR DESCRIPTION
**Pull request by zfrank7777**
The content is the basic structure of the back-end, showed in simple html websites.

### CODE REVIEW

**Comment by amjltc295** [comment]
__pycache__ files should not be in the repo

**Comment by zfrank7777** [comment]
Files deleted.

File: src/README.md [hunk 1]
```diff
@@ -1 +1,26 @@
...
+```
+
+### Running the Program
+Under src, the following commands help you start the server of the website.
+```
+export FLASK_APP=surfing_penguin.py
+flask run
+```
+
+You should see the following content if it is correctly run.
```
Comment (amjltc295) [inline src/README.md]: it runs correctly?

**Comment by amjltc295** [comment]
There are still  src/surfing_penguin/__pycache__/__init__.cpython-36.pyc
Try to add __pycached__ in .gitignore.

File: src/surfing_penguin/routes.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+def login():
+    form = LoginForm()
+    if form.validate_on_submit():
+        flash('Login requested for user {}, remember_me={}'.format(
+            form.username.data, form.remember_me.data))
+        return redirect(url_for('index'))
+    return render_template('login.html', title='Sign In', form=form)
+
+@surfing_penguin.route('/select_qstnr')
+def select_qstnr():
```
Comment (amjltc295) [inline src/surfing_penguin/routes.py]: Docstrings should be added.
Please see http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

File: src/surfing_penguin/__init__.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+from flask import Flask
```
Comment (amjltc295) [inline src/surfing_penguin/__init__.py:3]: Headers should be added as well.
See https://stackoverflow.com/questions/1523427/what-is-the-common-header-format-of-python-files

File: src/surfing_penguin/templates/select_qstnr.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+
+{% block content %}
```
Comment (amjltc295) [inline src/surfing_penguin/templates/select_qstnr.html:3]: What are these blocks for?

**Review by amjltc295 — COMMENTED** [review]
There are some minor style errors to be improved. Others look good.

File: src/surfing_penguin/templates/select_qstnr.html (hunk 4 above)
Comment (zfrank7777) [reply src/surfing_penguin/templates/select_qstnr.html:3]: For consistency, base.html contains the shared content of all html templates, and it has a block region for other templates to store their individual content. Each template based on it just fills in the block content.

File: src/surfing_penguin/routes.py [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+def index():
+    return render_template('index.html')
+
+@surfing_penguin.route('/login', methods=['GET', 'POST'])
+def login():
+    form = LoginForm()
+    if form.validate_on_submit():
+        flash('Login requested for user {}, remember_me={}'.format(
+            form.username.data, form.remember_me.data))
+        return redirect(url_for('index'))
```
Comment (amjltc295) [inline src/surfing_penguin/routes.py:20]: url_for is undefined.

File: src/surfing_penguin/__init__.py [hunk 6]
```diff
@@ -1,3 +1,5 @@
+"""__init__.py: Initialization of the server"
```
Comment (amjltc295) [inline src/surfing_penguin/__init__.py]: Miss quotes at the end.

**Comment by amjltc295** [comment]
Python files do not pass Flake8 style check.

**Review by amjltc295 — COMMENTED** [review]
Python files do not pass Flake8 style check.

**Comment by amjltc295** [comment]
Python files do not pass Flake8 style check. 
Install flake8 (https://pypi.python.org/pypi/flake8) and run `flake8 .`,
or add flake8 to Vim Syntastic plugin (set `let g:syntastic_python_checkers=['pyflakes', 'flake8']` in ~/.vim_runtime/vimrc/plugins_config.vim if you install amix/vimrc.

**Pull request merged**